src/site/content/en/index.md

```diff
@@ -1,25 +1,62 @@
 ---
-layout: homepage
-description: |
-  Get the web's modern capabilities on your own sites and apps with useful
-  guidance and analysis from web.dev.
-date: 2018-11-05
-i18n:
-  headline: Let's build the future of the web
-  subheadline: |
-    Get the web's modern capabilities on your own sites and apps with useful
-    guidance and analysis from web.dev.
-  test_cta: Test my site
-  about_subheadline: |
-    As the web advances, users' expectations grow. With web.dev's guidance, you
-    can give your users the best experience, wherever they are.
-  about_cta: About web.dev
-  explore_cta: Explore topics
-  latest_headline: Latest stories
-  latest_subheadline: See our latest news, updates, and stories for developers.
-  see_all: See all articles
-  explore_subheadline: |
-    Learn more about a topic, and practice your skills with a hands-on
-    codelab.
-  see_topics: See all topics
+layout: 'homepage-next'
+title: 'web.dev'
+intro:
+  eyebrow: 'About web.dev'
+  title: "Let's build the future of the web, together"
+  summary: |
+    'Take advantage of the latest modern technologies to build amazing web experiences for everyone'
```

Comment:
```suggestion
    'Take advantage of the latest modern technologies to build amazing web experiences for everyone.'
```

Reply:
These single quotes are displayed on the home page

Reply:
fixed by Sam